modules/nw-metalLB-upgrading-operator.adoc

```diff
@@ -0,0 +1,71 @@
+// Module included in the following assemblies:
+//
+// * networking/metallb/metallb-upgrading-operator.adoc
+
+:_content-type: PROCEDURE
+
+[id="upgrading-metalLB-configured-for-a-specific-namespace_{context}"]
+= Upgrading MetalLB when previously installed to a specific namespace
+
+Follow this procedure to allow upgrade if the Operator has previously been installed to a specific namespace.
+
+.Prerequisites
+
+* Install the {product-title} CLI (oc).
+* Log in to the OpenShift cluster as a user with `cluster-admin` privileges.
+
+.Procedure
+
+. Determine the unique name associated with the OperatorGroup:
++
+[source,terminal]
+----
+$ oc describe -n metallb-system og metallb-system
+----
++
+.Example output
++
+[source,terminal]
+----
+Name:         metallb-system-6m6ck
+Namespace:    metallb-system
+Labels:       <none>
+Annotations:  olm.providedAPIs:
+                AddressPool.v1alpha1.metallb.io,AddressPool.v1beta1.metallb.io,BFDProfile.v1beta1.metallb.io,BGPPeer.v1beta1.metallb.io,MetalLB.v1beta1.me...
+API Version:  operators.coreos.com/v1
+Kind:         OperatorGroup
+Metadata:
+  Creation Timestamp:  2022-06-30T09:15:41Z
+  Generate Name:       metallb-system-
+  Generation:          1
+  Managed Fields:
+    API Version:  operators.coreos.com/v1
+    Fields Type:  FieldsV1
+    fieldsV1:
+      f:metadata:
+        f:generateName:
+      f:spec:
+        .:
+        f:targetNamespaces:
+          .:
+          v:"metallb-system":
+...
+----
+
+. Edit the MetalLB `OperatorGroup` CR and remove the `spec` element that contains the `targetNamespaces` entry by running the following command:
++
+[source,terminal]
+----
+$ oc patch operatorgroup -n metallb-system metallb-system-6m6ck --type json -p '[{ "op": "remove", "path": "/spec" }]'
+----
+
+. Wait until the Operator Lifecycle Manager (OLM) processes the change.
+
+. Verify that the OperatorGroup CR change has been successfully applied. Check that the `OperatorGroup` CR `spec` element has been removed:
++
+[source,terminal]
+----
+$ oc describe -n metallb-system og metallb-system
+----
```

Comment:
Could you include the output of describe so that customer understands the change.